lldb/source/Plugins/TypeSystem/Swift/SwiftASTContext.cpp

```diff
@@ -5876,13 +5876,7 @@ CompilerType SwiftASTContext::GetPointeeType(opaque_compiler_type_t type) {
 
 CompilerType SwiftASTContext::GetPointerType(opaque_compiler_type_t type) {
   VALID_OR_RETURN(CompilerType());
-
-  if (type) {
-    swift::Type swift_type(GetSwiftType({this, type}));
-    const swift::TypeKind type_kind = swift_type->getKind();
-    if (type_kind == swift::TypeKind::BuiltinRawPointer)
-      return ToCompilerType({swift_type});
-  }
+  assert(false && "GetPointerType is unimplemented for Swift types");
```

Comment:
Should I use Swift's logging system, or print directly to `stderr`?

Reply:
A print to stderr should be reserved for conditionally-compiled debug-only code (`llvm::dbgs()`) — usually stderr is a dead end if we are, e.g., running in Xcode. `LLDB_LOG()` is the preferred method.

Reply:
thanks!

Reply:
> Should I use Swift's logging system

I meant to say *lldb*'s logging system